### PR TITLE
Remove GA feature gates in 1.28 about DryRun

### PR DESCRIPTION
**Pull request by yulng**
Remove GA feature gates in 1.28 about DryRun
/kind cleanup
Thanks

### CODE REVIEW

File: test/integration/dryrun/dryrun_test.go [hunk 1]
```diff
@@ -211,7 +210,7 @@ func DryRunDeleteTest(t *testing.T, rsc dynamic.ResourceInterface, name string)
 
 // TestDryRun tests dry-run on all types.
 func TestDryRun(t *testing.T) {
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DryRun, true)()
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, "", true)()
```
Comment (wzshiming) [inline test/integration/dryrun/dryrun_test.go:213]:
```suggestion
```

Please update the relevant tests

**Review by wzshiming — COMMENTED** [review]
/ok-to-test

**Comment by yulng** [comment]
@liggitt 
hi, liggitt
I have applied for kubernetes membership. Could you give me +1
https://github.com/kubernetes/org/issues/4194
Thank you very much

**Pull request merged**